puppet/modules/user_tools/manifests/init.pp

```diff
@@ -143,6 +143,7 @@
 
 10.1.1.11 i1 igus1
```

Comment:
Now that you don't use the darwins anymore you could move these to the much coveted 10.1.1.robotnumber position

Reply:
I thought about that, figured we should maintain some legacy for a little longer